src/sentry/migrations/0648_create_sentryshot.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 5.0.2 on 2024-02-20 20:52
+
+import uuid
+
+import django.utils.timezone
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.hybrid_cloud_foreign_key
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0647_apitoken_add_hashed_columns"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="SentryShot",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                ("uuid", models.UUIDField(default=uuid.uuid4, editable=False)),
```

Comment:
If you're going to do reads/queries on this column you'll want an index on it to avoid a table scan.